block.json

```diff
@@ -0,0 +1,74 @@
+{
+	"name": "tenup/maps-block-apple",
+	"title": "Apple Maps",
+	"category": "location",
```

Comment:
This category appears to be a custom one, maybe we should update to `embeds` to use one of the default block categories? cc: @fabiankaegy

Reply:
Looks like `embed` is the correct spelling based on https://github.com/WordPress/gutenberg/blob/master/docs/designers-developers/developers/block-api/block-registration.md#category